athena/tests/modules/modeling/module_modeling_llm/mock/conftest.py

```
@@ -1,115 +1,47 @@
-    return MockLanguageModel()
 
+class FakeModelConfig(OpenAIModelConfig):
+    """A fake ModelConfig that extends OpenAIModelConfig for testing"""
 
-@pytest.fixture
-def mock_assessment_model():
-    """Fixture providing a mock assessment model."""
-    return MockAssessmentModel()
+    model_name: str = "fake_model"
 
+    def get_model(self, openai_catalog=None):
```

Ruff (0.13.3)

Unused method argument: `openai_catalog`

(ARG002)